packages/core/src/components/Link/Link.stories.tsx

```diff
@@ -73,7 +73,7 @@ export const PassProps = () => {
       <Link
         to={routeRef.path}
         /** react-router-dom related prop */
-        component={RouterNavLink}
+        as={RouterNavLink}
```

Comment:
Is this a prop that is passed to our link component, through to the MUI link component, and finally to the react-router link component, to render with another link component?

Reply:
Ah, sorry. Yeah this is why I got all confused. This should be reverted back to `component` again. Will fix now.